package.json

```diff
@@ -90,25 +90,19 @@
       "/.module-cache/",
       "/react/build/"
     ],
-    "persistModuleRegistryBetweenSpecs": true,
     "rootDir": "",
     "scriptPreprocessor": "scripts/jest/preprocessor.js",
-    "setupEnvScriptFile": "scripts/jest/environment.js",
+    "setupFiles": [ "scripts/jest/environment.js" ],
     "setupTestFrameworkScriptFile": "scripts/jest/test-framework-setup.js",
-    "testFileExtensions": [
-      "coffee",
-      "js",
-      "ts"
-    ],
+    "testRegex": "/__tests__/",
+    "moduleFileExtensions": ["js", "json", "node", "coffee", "ts"],
```

Comment:
Note that js, json and node are the default ones (that are needed to run the tests) and we need in addition coffee and ts for our integration testing with those.